codes/3_coding.py

```
@@ -122,19 +124,11 @@ def get_write_msg(todo_file_name, detailed_logic_analysis, done_file_lst):
-        completion = client.chat.completions.create(
-            model=gpt_version, 
-            reasoning_effort="high",
-            messages=msg
-        )
-    else:
-        completion = client.chat.completions.create(
-            model=gpt_version, 
-            messages=msg
-        )
-    return completion
+async def api_call(msg):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)